diff --git a/src/Command/CivicrmPublishCommand.php b/src/Command/CivicrmPublishCommand.php
index d854136..1dc5442 100644
--- a/src/Command/CivicrmPublishCommand.php
+++ b/src/Command/CivicrmPublishCommand.php
@@ -21,7 +21,7 @@ class CivicrmPublishCommand extends \Composer\Command\BaseCommand {
       ->addOption('delete', 'D', InputOption::VALUE_NONE, 'Enable broad deletion. This ensures that orphaned files are removed, but it requires more I/O.');
   }
 
-  protected function execute(InputInterface $input, OutputInterface $output) {
+  protected function execute(InputInterface $input, OutputInterface $output): int {
     $fileMode = $input->getOption('file-mode');
     if ($fileMode && in_array($fileMode, ['auto', 'copy', 'symlink', 'symdir'])) {
       putenv('CIVICRM_COMPOSER_ASSET=' . $fileMode);
